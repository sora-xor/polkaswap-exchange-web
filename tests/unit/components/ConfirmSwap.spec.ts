// import { shallowMount, createLocalVue } from '@vue/test-utils'
// import Vuex from 'vuex'

// import ConfirmSwap from '@/components/ConfirmSwap.vue'
// import { tokens } from '../../mocks/tokens'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// localVue.use(Vuex)
// SoramitsuElementsImport(localVue)

// describe('ConfirmSwap.vue', () => {
//   let getters
//   let store

//   beforeEach(() => {
//     TranslationMock(ConfirmSwap)

//     getters = {
//       tokenFrom: () => tokens[0],
//       tokenTo: () => tokens[1],
//       fromValue: () => 150.123654,
//       toValue: () => 0
//     }

//     store = new Vuex.Store({
//       getters
//     })
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(ConfirmSwap, {
//       localVue,
//       store,
//       propsData: {
//         visible: true,
//         title: 'Confirm Swap',
//         customClass: 'dialog--confirm-swap'
//       }
//     })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('ConfirmSwap test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
