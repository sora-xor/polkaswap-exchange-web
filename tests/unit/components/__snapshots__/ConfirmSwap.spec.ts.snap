// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmSwap.vue should renders correctly 1`] = `
<dialog-base-stub
  customclass="dialog--confirm-swap"
  title="Confirm Swap"
>
  <div
    class="tokens"
  >
    <div
      class="tokens-info-container"
    >
      <span
        class="token-value"
      >
        150.1237
      </span>
       
      <s-icon-stub
        class="icon-divider"
        name="arrow-bottom-rounded"
        size="medium"
      />
       
      <span
        class="token-value"
      >
        0.0000
      </span>
    </div>
     
    <div
      class="tokens-info-container"
    >
      <div
        class="token"
      >
        <!---->
        
        XOR
      
      </div>
       
      <div
        class="token"
      >
        <!---->
        
        KSM
      
      </div>
    </div>
  </div>
   
  <p
    class="transaction-message"
  />
   
  <s-divider-stub
    contentposition="center"
    direction="horizontal"
    type="secondary"
  />
   
  <!---->
   
  <!---->
</dialog-base-stub>
`;
